### PR TITLE
chore(ckbtc): Proposals to upgrade ckBTC ledger suite

### PR DESCRIPTION
**Pull request by gregorydemay**
Proposals to upgrade the ckBTC ledger suite (index, ledger and archive canisters) to `rc--2024-09-14_01-30` (commit `d4ee25b0865e89d3eaac13a60f0016d5e3296b31`)

### CODE REVIEW

File: rs/bitcoin/ckbtc/mainnet/ckbtc_archive_upgrade_2024_09_27.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Proposal to upgrade the ckBTC archive canister
+
+Git hash: `d4ee25b0865e89d3eaac13a60f0016d5e3296b31`
+
+New compressed Wasm hash: `9476aa71bcee621aba93a3d7c115c543f42c543de840da3224c5f70a32dbfe4d`
```
Comment (ninegua) [inline rs/bitcoin/ckbtc/mainnet/ckbtc_archive_upgrade_2024_09_27.md:5]: checked

File: rs/bitcoin/ckbtc/mainnet/ckbtc_index_upgrade_2024_09_27.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Proposal to upgrade the ckBTC index canister
+
+Git hash: `d4ee25b0865e89d3eaac13a60f0016d5e3296b31`
+
+New compressed Wasm hash: `612410c71e893bb64772ab8131d77264740398f3932d873cb4f640fc257f9e61`
```
Comment (ninegua) [inline rs/bitcoin/ckbtc/mainnet/ckbtc_index_upgrade_2024_09_27.md:5]: checked

File: rs/bitcoin/ckbtc/mainnet/ckbtc_ledger_upgrade_2024_09_27.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Proposal to upgrade the ckBTC ledger canister
+
+Git hash: `d4ee25b0865e89d3eaac13a60f0016d5e3296b31`
+
+New compressed Wasm hash: `a170bfdce5d66e751a3cc03747cb0f06b450af500e75e15976ec08a3f5691f4c`
```
Comment (ninegua) [inline rs/bitcoin/ckbtc/mainnet/ckbtc_ledger_upgrade_2024_09_27.md:5]: checked.

**Review by ninegua — APPROVED** [review]

File: rs/bitcoin/ckbtc/mainnet/ckbtc_archive_upgrade_2024_09_27.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## Wasm Verification
+
+Verify that the hash of the gzipped WASM matches the proposed hash.
+
+```
+git fetch
+git checkout d4ee25b0865e89d3eaac13a60f0016d5e3296b31
+./ci/container/build-ic.sh -c
+sha256sum ./artifacts/canisters/ic-icrc1-archive.wasm.gz
```
Comment (THLO) [inline rs/bitcoin/ckbtc/mainnet/ckbtc_archive_upgrade_2024_09_27.md:42]:
```
$ sha256sum ./artifacts/canisters/ic-icrc1-archive.wasm.gz
9476aa71bcee621aba93a3d7c115c543f42c543de840da3224c5f70a32dbfe4d  ./artifacts/canisters/ic-icrc1-archive.wasm.gz
```

Confirmed!

File: rs/bitcoin/ckbtc/mainnet/ckbtc_index_upgrade_2024_09_27.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+## Wasm Verification
+
+Verify that the hash of the gzipped WASM matches the proposed hash.
+
+```
+git fetch
+git checkout d4ee25b0865e89d3eaac13a60f0016d5e3296b31
+./ci/container/build-ic.sh -c
+sha256sum ./artifacts/canisters/ic-icrc1-index-ng.wasm.gz
```
Comment (THLO) [inline rs/bitcoin/ckbtc/mainnet/ckbtc_index_upgrade_2024_09_27.md:44]:
```
$ sha256sum ./artifacts/canisters/ic-icrc1-index-ng.wasm.gz
612410c71e893bb64772ab8131d77264740398f3932d873cb4f640fc257f9e61  ./artifacts/canisters/ic-icrc1-index-ng.wasm.gz
```

Confirmed!

File: rs/bitcoin/ckbtc/mainnet/ckbtc_ledger_upgrade_2024_09_27.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+## Wasm Verification
+
+Verify that the hash of the gzipped WASM matches the proposed hash.
+
+```
+git fetch
+git checkout d4ee25b0865e89d3eaac13a60f0016d5e3296b31
+./ci/container/build-ic.sh -c
+sha256sum ./artifacts/canisters/ic-icrc1-ledger.wasm.gz
```
Comment (THLO) [inline rs/bitcoin/ckbtc/mainnet/ckbtc_ledger_upgrade_2024_09_27.md:52]:
```
$ sha256sum ./artifacts/canisters/ic-icrc1-ledger.wasm.gz
a170bfdce5d66e751a3cc03747cb0f06b450af500e75e15976ec08a3f5691f4c  ./artifacts/canisters/ic-icrc1-ledger.wasm.gz
```

Confirmed!

**Review by THLO — APPROVED** [review]

**Pull request merged**